### DISCUSSION TITLE
How to add keyboard shortcut in typer as command?

### DISCUSSION THREAD

**Discussion by leamandeep** [fastapi/typer#650] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.command()
def settings:
   // settings code
```

### Description

I want to add a keyboard shortcut to my cli app. I don't how to do it in Typer.  I want to use a keyboard shortcut `ctrl+shift+s` to use my setting functions. Any guidance on how to do it?

### Operating System

Linux

### Typer Version

0.9.0

### Python Version

3.11.4

**Comment by Apocryphon-X** [comment, marked as answer]
Based on the description you've given, I don't think that's something Typer should take care of. In any case, you might be looking for something like the monitoring capabilities of `pyinput` or the TUI interfaces of `textual` with the key combinations of your preference (such as `ctrl+shift+s`).

Edit: I forgot about [`keyboard`](https://github.com/boppreh/keyboard) (in this context it's easier to use than the other alternatives).

**Reply by leamandeep** [reply]
yes, I'm looking for that. Thanks for guiding me
